src/config/dynamic-theme-fixes.config

```diff
@@ -8356,6 +8356,15 @@ CSS
 
 ================================
 
+icrc.org
+
+CSS
+.logo, .logo img {
```

Comment:
I don't think `.logo img` does need this fix as well, just applying it to `.logo` should be sufficient.

Reply:
`.logo img` is still needed on most pages (for example, https://www.icrc.org/en/what-we-do/covid-19-pandemic).

Reply:
```suggestion
.logo,
.logo img {
```